Documentation/architecture/net-platform-standard.md

```diff
@@ -134,13 +134,21 @@ Existing PCL projects in VS2013 and VS2015 (excluding UWP targets), can only tar
 
 ### Deprecated monikers
 
-| Platform | NuGet identifier |
-| ---------| --------------- |
-| ASP.NET 5.0 on .NET Framework | aspnet50 |
-| ASP.NET 5.0 on .NET Core | aspnetcore50 |
-| DNX on .NET Framework 4.5.1 - 4.6 | dnx451 - dnx46 |
-| DNX on .NET Core 5.0 | dnxcore50  |
-| Windows 8 | winrt |
+| Platform | Deprecated NuGet identifier | Current NuGet identifier |
+| ---------| --------------------------- | ------------------------ |
+| ASP.NET 5.0 on .NET Framework | aspnet50 | net46 |
+| ASP.NET 5.0 on .NET Core | aspnetcore50 | netstandard1.3 |
+| DNX on .NET Framework 4.5.1 - 4.6 | dnx451 - dnx46 | net451 - net46 |
+| DNX on .NET Core 5.0 | dnxcore50  | dotnet &rarr; dotnet5.4 &rarr; netstandard1.3 |
```

Comment:
> There might be TFMs, that contain dnxcore50, which target different netstandard versions

In all cases DNXCore50 was a super-set of previous portable profiles, so adding it to the portable-* list wasn't actually representing anything different surface-area-wise than what was already there.  It was a hack to try to get NuGet to install the package in DNXCore50 project.  It can effectively be ignored when doing the mapping.  Another way to think about the mapping is to look at each entry in the portable-a+b+c clause and use the minimum netstandard version supported by all of them.

>  is it bad practice to just do an `"imports": "dnxcore50"`? 

It's always bad practice to use `"imports":`.  Imports is a stop-gap until folks update packages with support for specific platforms.  It should really only be used in applications and even then it *voids the warranty* provided by the package author.  The package author told you they'll work on X & Y, but you're saying you want to use it on Z and just try to use the X implementation. You may be missing dependencies, may hit PInvokes that won't work on Z, etc.
If you use `imports` in a library, then build a package from that library, all of your package's consumers would have to use imports as well.  So it should never be used in this circumstance.